QuizNet/Controllers/QuestionController.cs

```diff
@@ -67,16 +68,32 @@ public IActionResult Save(QuestionFormViewModel viewModel)
             }
             else
             {
+                question.CreationTime = DateTime.Now;
```

Comment:
Generalnie ustawianie daty w tym miejscu działa, ale zastanawiam się czy nie istnieje jakieś lepsze miejsce do tego.
@KrzysztofMilewski masz jakąś sugestię?
Jakbym to robił ja to pewnie datę ustawiałbym w serwisie ewentualnie jeszcze niżej - na poziomie repozytorium.

Reply:
Zgadzam się z Danielem - kontroler nie jest miejscem, w którym takie rzeczy (związane de facto z modelem bazodanowym) powinny być wykonywane. Ja prawdopodobnie przeniósłbym ustawianie daty do serwisu, a do repozytorium przekazywał już gotowy do wstawienia obiekt.

Reply:
Tak na początku zrobiłem, ale miałem wątpliwości - czy metoda w serwisie zawsze będzie używana tylko do tworzenia pytania i nie będzie używana w jakimś innym przypadku dodawania, niezwiązanym z datą? Jeżeli tak, to czy lepiej wtedy stworzyć oddzielną metodę w serwisie?

Reply:
Metoda Add w serwisie jest używana tylko do dodawania pytań. Nie powinna być używana w żadnym innym celu. S w SOLID. Jaką proponujesz oddzielna metodę w serwisie?

Reply:
Coś w stylu `CreateQuestion(QuestionDto question)`, ustawiałaby wtedy `question.CreationTime`, a potem dodała `question` przez repozytorium. W sumie robi to samo co Add, ale chodzi mi o to rozgraniczenie. Do Add można wrzucić pytanie z jakąś dowolną datą.

Reply:
Według mnie nie trzeba tak rozgraniczac

Reply:
Szczerze mówiąc nie do końca rozumiem, co masz na myśli mówiąc "inny przypadek dodawania niezwiązany z datą" - pole w klasie Question ma nazwę CreationTime, co sugeruje, że zawsze w przypadku tworzenia pytania należy ustawić datę. Ustawianie tej daty w metodze Add jest jak najbardziej OK

Reply:
W porządku

Reply:
Dodasz poprawki? Wtedy zaakceptuje PR :)